cli/cmd/sync.go

```diff
@@ -33,16 +34,22 @@ func NewCmdSync() *cobra.Command {
 	return cmd
 }
 
+func getPluralDirectories(count int) string {
```

Comment:
Why not something like: `func pluralize(count int, singular, plural string) string`